components/dashboard/src/projects/Project.tsx

```diff
@@ -382,7 +382,7 @@ export default function ProjectsPage() {
                                                                 ? []
                                                                 : [
                                                                       {
-                                                                          title: "New Workspace ...",
+                                                                          title: "New Workspace with ...",
```

Comment:
fwiw I think this additional option will go away shortly and we'll be navigating to a new start workspace page where options are always available. I see we already approved merging this wording change a few times, so don't want to block it.